<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserSelect</class>
 <widget class="QDialog" name="UserSelect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>850</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>850</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>20</y>
     <width>261</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>EA Sports Covers SC</family>
     <pointsize>45</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(48, 76, 135)
</string>
   </property>
   <property name="text">
    <string>Select User</string>
   </property>
  </widget>
  <widget class="QListWidget" name="userList">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>220</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="newUser">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>220</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>150</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>EA Sports Covers SC</family>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(48, 76, 135)
</string>
   </property>
   <property name="text">
    <string>Existing</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>150</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>EA Sports Covers SC</family>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(48, 76, 135)
</string>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>490</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;</string>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>490</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="existingCheck">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>170</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="newCheck">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>170</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>UserSelect</receiver>
   <slot>onCancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>323</x>
     <y>508</y>
    </hint>
    <hint type="destinationlabel">
     <x>159</x>
     <y>505</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>UserSelect</receiver>
   <slot>onSubmit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>496</x>
     <y>509</y>
    </hint>
    <hint type="destinationlabel">
     <x>684</x>
     <y>505</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onSubmit()</slot>
  <slot>onCancel()</slot>
 </slots>
</ui>
